services/shhext/service.go

```diff
@@ -213,6 +213,7 @@ func (s *Service) retrieveMessagesLoop(tick time.Duration, cancel <-chan struct{
 							Message: message.TransportMessage,
 						}
 						dedupMessage.Message.Payload = message.DecryptedPayload
```

Comment:
Do we need this then?

Reply:
yes, we need still timestamp, but working on related code (I might need to make some other changes), in the next PR I will just drop that and add the `timestamp` field directly.